tests/testlib/s2n_kem_tests.c

```diff
@@ -15,22 +15,14 @@
 
 #include "tls/s2n_kem.h"
 #include "tests/testlib/s2n_nist_kats.h"
+#include "tests/testlib/s2n_testlib.h"
 #include "utils/s2n_mem.h"
 #include "utils/s2n_random.h"
 #include "utils/s2n_safety.h"
 #include "crypto/s2n_fips.h"
 
 #define SEED_LENGTH 48
 uint8_t kat_entropy_buff[SEED_LENGTH] = {0};
-struct s2n_blob kat_entropy_blob = {.size = SEED_LENGTH, .data = kat_entropy_buff};
```

Comment:
The drbg bits in `s2n_hybrid_kem_tests.c` are wrapped in `#if S2N_LIBCRYPTO_SUPPORTS_CUSTOM_RAND`, but the drbg bits in this file are not. I'm wondering if that was a previous miss on our part, and if it affects this refactor at all.

Reply:
Just pushed an update that addresses this: https://github.com/awslabs/s2n/pull/1919/commits/3a5f6813f966bd73bfd644dff184973ebd266cb8

"We can set the drbg when !S2N_LIBCRYPTO_SUPPORTS_CUSTOM_RAND, it
just won't be used by libcrypto methods." I *think* this is the explanation.